packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -60,7 +60,66 @@ export class WaferMap extends FoundationElement {
         values: []
     };
 
-    private readonly dataManager: DataManager | undefined;
+    private renderQueued = false;
+
+    private dataManager: DataManager | undefined;
+
+    private quadrantChanged(): void {
+        this.queueRender();
+    }
+
+    private orientationChanged(): void {
+        this.queueRender();
+    }
+
+    private maxCharactersChanged(): void {
+        this.queueRender();
+    }
+
+    private dieLabelsSuffixChanged(): void {
+        this.queueRender();
+    }
+
+    private colorsScaleModeChanged(): void {
```

Comment:
I was initially confused by the inconsistent naming of `colorScale` and `colorsScaleMode` (additional "s" after color). The names of the change functions are correct in that they match the property name, but we should pick one format (I'd vote no "s") and update the property names and change functions to match.

This doesn't have to happen in this PR.